templates/v2/vault/nav/_nav.html.twig

```diff
@@ -0,0 +1,38 @@
+{% set isBeneficiary = app.user.isBeneficiaire %}
+{% set itemsList = {
```

Comment:
I feel like this should be generated in PHP, like what is done in KNP Menu bundle